src/components/Notification/index.js

```diff
@@ -0,0 +1,43 @@
+import { h, Component } from 'preact';
+import PropTypes from 'prop-types';
+
+import { applyTheme } from '../ThemeProvider/';
+
+import './style.css';
+
+/**
+* Notification
+*
+*/
+class Notification extends Component {
+  propTypes = {
+    theme: PropTypes.string,
+    network: PropTypes.string,
+  };
+
+  render () {
+    const { theme, network } = this.props;
+    if (network === '') return null;
+
+    const renderNotification = () => {
+      switch (network) {
+        case 'disconnected':
+          return 'Chat Disconnected';
+        case 'reconnecting':
+          return 'Disconnected; trying to reconnect...';
+        case 'reconnected':
+          return 'Disconnected; trying to reconnect...';
+        default:
+          return '';
+      }
+    };
```

Comment:
yeah probably.

Also, btw, make sure you comment under the line you are referring to when you make a PR so that I know what you're talking about :)